cmd/podman/top.go

```diff
@@ -53,12 +53,26 @@ func init() {
 	topCommand.SetHelpTemplate(HelpTemplate())
 	topCommand.SetUsageTemplate(UsageTemplate())
 	flags := topCommand.Flags()
+	flags.SetInterspersed(false)
 	flags.BoolVar(&topCommand.ListDescriptors, "list-descriptors", false, "")
 	flags.MarkHidden("list-descriptors")
 	flags.BoolVarP(&topCommand.Latest, "latest", "l", false, "Act on the latest container podman is aware of")
 	markFlagHiddenForRemoteClient("latest", flags)
 }
 
+func genDescriptors(args []string) []string {
```

Comment:
Uh. Do we really want to do this? Some of those options do things...

Might be safer to add `-e` and `-o` as NOOPs.

Reply:
Docker supports a bunch that we do not.  I think we should just ignore for now, and then let @vrothberg work on a better solution in the long run.

Reply:
Basically Docker allows you to pass any option that the ps command supports.

Reply:
`-o` might actually be a good idea to implement as its own flag - it specifies that format specifiers will follow in comma-separated format, so different than our default.

Potential problem: I don't know if Cobra allows mixing flags with arguments and flags without for single-character flags, so `-eo` might not work together.

`-e` is a NO-OP with our current implementation, we always do what it specifies.

Reply:
Huh. Cobra does allow it.
`podman run --rm -tiv /tmp:/tmp:rw fedora bash` works

Reply:
@mheon we already support the -o option by default.  If you specify any descriptors after the top command it will use that option.  This patch just ignores the user if he specified -* and allows Descriptors to be comma or space separate.

This patch just allows the following syntax.
```
$ ./bin/podman top dd7ba27b pid hpid label
PID   HPID    LABEL
1     32237   system_u:system_r:container_t:s0:c419,c693
$ ./bin/podman top dd7ba27b pid,hpid,label
PID   HPID    LABEL
1     32237   system_u:system_r:container_t:s0:c419,c693
$ ./bin/podman top dd7ba27b -eo pid,hpid,label
PID   HPID    LABEL
1     32237   system_u:system_r:container_t:s0:c419,c693
```

Reply:
That's not how `docker top -o` works - it'd be `docker top -o pid,hpid,label` - note the commas

Reply:
Yes, I know, we have different options in podman top, I am just trying to make sure we can handle basic `docker top` commands but still support our own versions.

Reply:
We are more lax about the syntax and not tied to the way the ps command handles it.

Reply:
We can pretty easily implement `-o` as a flag that supports comma-separated format specifiers, though

Reply:
I am not sure,but I think Docker might require the ps command inside of the container.  We do not since @vrothberg wrote a native go implementation psgo.

https://github.com/containers/psgo

Reply:
We don't have to, this patch allows that syntax.